book/dev-env/iOS.md

```diff
@@ -47,6 +54,7 @@ end
 ```
 
 This Podfile references the podspec and will create a structure where the plugin code is imported like a real project as a development pod.
+*Note:* The example above uses Swift 4.2. Please choose 4.2 or 5.0 according to the Zapp SDK version you are using for development as mentioned at the top of this document.
```

Comment:
source 'git@github.com:CocoaPods/Specs.git' has changed to 
source 'https://cdn.cocoapods.org/'.

ref: https://blog.cocoapods.org/CocoaPods-1.7.2/

Reply:
👍